### PR TITLE
[MI-3655]: Added test cases for message edit feature in MS Teams

### PR DESCRIPTION
**Pull request by AayushChaudhary0001**
This PR consist of the test cases of message edit feature:-

- Text and emoji edit
- Mentions edit

### CODE REVIEW

File: data/test-cases/plugins/MS Teams sync/Message edit/Text_and_emoji_edit.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+**Step 1**
+
+1. Enter any text/emoji on MM channel(already linked) or DM/GM.
+2. Edit the message on the same MM channel(already linked) or DM/GM with any other text meesage/emoji.
+
+**Expected**
+
+The desired message shoiuld be edited and get updated on the Teams side as well.
```
Comment (ayusht2810) [inline data/test-cases/plugins/MS Teams sync/Message edit/Text_and_emoji_edit.md:43]:
```suggestion
The desired message should be edited and updated on the Teams side as well.
```

File: data/test-cases/plugins/MS Teams sync/Message edit/Text_and_emoji_edit_2.md
Comment (ayusht2810) [inline data/test-cases/plugins/MS Teams sync/Message edit/Text_and_emoji_edit_2.md:1]: can't we append the suffixes `teams_to_mm` and `mm_to_teams` in both files?

File: data/test-cases/plugins/MS Teams sync/Message edit/Text_and_emoji_edit_2.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Enter any text/emoji on Teams channel(already linked) or DM/GM.
+2. Edit the message on the same Teams channel(already linked) or DM/GM with any other text meesage/emoji.
```
Comment (ayusht2810) [inline data/test-cases/plugins/MS Teams sync/Message edit/Text_and_emoji_edit_2.md:39]:
```suggestion
2. Edit the message on the same Teams channel(already linked) or DM/GM with any other text message/emoji.
```

File: data/test-cases/plugins/MS Teams sync/Message edit/Text_and_emoji_edit.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Enter any text/emoji on MM channel(already linked) or DM/GM.
+2. Edit the message on the same MM channel(already linked) or DM/GM with any other text meesage/emoji.
```
Comment (ayusht2810) [inline data/test-cases/plugins/MS Teams sync/Message edit/Text_and_emoji_edit.md:39]:
```suggestion
2. Edit the message on the same MM channel(already linked) or DM/GM with any other text message/emoji.
```

File: data/test-cases/plugins/MS Teams sync/Message edit/Text_and_emoji_edit_2.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+**Expected**
+
+The desired message shoiuld be edited and get updated on the MM side as well.
```
Comment (ayusht2810) [inline data/test-cases/plugins/MS Teams sync/Message edit/Text_and_emoji_edit_2.md:43]:
```suggestion
The desired message should be edited and updated on the MM side as well.
```

**Review by ayusht2810 — APPROVED** [review]

File: data/test-cases/plugins/MS Teams sync/Message edit/Mentions_edit.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+2. Edit the mentioned user and mention any other user.
+
+**Step 2**
+
+1. Enter any message in a channel(already linked) or in DM/GM by mentioning any user in Teams.
+2. Edit the mentioned user and mention any other user.
+
+**Expected**
+
+The desired message should be edited and the desired user should be mentioned in the edited message in Teams. Also in step 2, the desired message should be edited and the desired user should be mentioned in the edited message in MM.
```
Comment (manojmalik20) [inline data/test-cases/plugins/MS Teams sync/Message edit/Mentions_edit.md:48]:
```suggestion
The desired message should be edited and the updated user should be mentioned in the edited message in Teams. Also in step 2, the desired message should be edited and the updated user should be mentioned in the edited message in MM.
```

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**